frontend/ui/uimanager.lua

```diff
@@ -825,6 +825,23 @@ function UIManager:unsetRunForeverMode()
     self._gated_quit = self.quit
 end
 
+-- Ignore an empty window stack *once*; for startup w/ a missing last_file shenanigans...
+function UIManager:runForOneFrame()
```

Comment:
(This method name sounds a bit cryptic/ambiguous to me, one frame = one framebuffer screen, one time slot, but we don't really use that word "frame" in the rest of UIManager (except in a few of your comments, "UI frame". If we have runForever, a :runOnce() would sound obvious, but I'm not sure it's really what this does :)

Reply:
`runOnce` sounds good to me ;).